src/main/scala/com/datastax/gatling/plugin/utils/CqlPreparedStatementUtil.scala

```diff
@@ -857,4 +902,14 @@ object CqlPreparedStatementUtil extends CqlPreparedStatementUtil {
         throw new CqlTypeException(s"$paramName expected to be type of ByteBuffer, Array[Byte] or Byte")
     }
   }
+
+  def toLocalDate(epochMillis: Long): LocalDate = {
+    val end = Instant.ofEpochMilli(epochMillis)
+    val d = Duration.between(Instant.EPOCH, end)
+    LocalDate.ofEpochDay(d.toDays)
+  }
+
+  def clzFromCodec(bindable: Bindable[_], genType: DataType): Class[Any] = {
+    bindable.codecRegistry().codecFor(genType).getJavaType.getRawType.asInstanceOf[Class[Any]]
```

Comment:
Return type and the asInstanceOf here is just to make the compiler happy.  The raw class type coming off of getRawType() is a superclass-bounded class instance but the Java type expects Class<T> for <T> defined at the method def.  If we just throw the superclass-bounded version at the set*() method scalac complains about a type mismatch... but they _are_ equivalent types underneath.  So we change to Class[Any] here to satisfy types at compile-time.